physical/raft/raft_test.go

```diff
@@ -203,6 +204,61 @@ func TestRaft_Backend(t *testing.T) {
 	physical.ExerciseBackend(t, b)
 }
 
+func TestRaft_Backend_LargeValue(t *testing.T) {
+	b, dir := getRaft(t, true, true)
+	defer os.RemoveAll(dir)
+
+	t.Helper()
+
+	value := make([]byte, defaultKVMaxValueSize+1)
+	rand.Read(value)
+	entry := &physical.Entry{Key: "foo", Value: value}
+
+	if err := b.Put(context.Background(), entry); err == nil {
```

Comment:
If you wrap the error, you can use `errwrap.ContainsType` or `errwrap.Contains` to ensure that the error returned is exactly what you want.